hack/order-prow-job-config.sh

```diff
@@ -7,6 +7,13 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-ci_operator_dir="$( dirname "${BASH_SOURCE[0]}" )/../ci-operator"
+base_dir="${1-}"
+
+if [[ -z "${base_dir}" ]] || ! [[ -d "${base_dir}" ]]; then
```

Comment:
nit: the point of using `[[` over `[` is that you can stay inside of one statement and not use bash logic along with `test` logic...


```sh
if [[ -z "${base_dir:-}" || ! -d "${base_dir}" ]]; then
```

Although you could really just get away with:

```sh
if [[ ! -d "${1:-}" ]]; then
```

Reply:
Thanks, addressed.